docs/source/dev-guide/plugins/index.rst

```diff
@@ -124,6 +124,16 @@ API
 For even more detailed information about our plugin system, please the see the
 :doc:`Plugin API </dev-guide/api/conda/plugins/index>` section.
 
+Error handling
+--------------
+
+Errors in ``conda`` are routed through :class:`conda.exception_handler.ExceptionHandler`, which can
+print additional information about the ``conda`` installation when an *unexpected* exception is
+found. These automatic reports can be really verbose and can get in the way of communicating
+*expected* errors. See `this issue in conda-build`_ as an example.
+
+To mark exceptions as *expected*, plugins should raise :class:`conda.CondaError` or a subclass
+thereof.
```

Comment:
Should we also include an example to an existing project? This might be helpful:

- https://github.com/conda-incubator/conda-auth/blob/main/conda_auth/exceptions.py

Reply:
```suggestion
To mark exceptions as *expected*, plugins should raise :class:`conda.CondaError` or a subclass
thereof. See [`conda_auth.exceptions`](https://github.com/conda-incubator/conda-auth/blob/0.2.1/conda_auth/exceptions.py) for an example.
```